packages/ui-byline/src/Byline/__tests__/Byline.test.tsx

```diff
@@ -82,53 +81,4 @@ describe('<Byline />', async () => {
     // @ts-expect-error ts-migrate(2304) FIXME: Cannot find name 'find'.
     expect(await find('figure')).to.exist()
   })
-
-  it(`should not allow the 'as' prop`, async () => {
-    const consoleError = stub(console, 'error')
-    // @ts-expect-error FIXME remove this line to see the error
-    await mount(<Byline as="foo">{image}</Byline>)
-    expect(consoleError).to.be.calledWith(
-      "Warning: [Byline] prop 'as' is not allowed."
-    )
-  })
-
-  describe('when passing down props to View', async () => {
-    const allowedProps = {
-      margin: 'small'
-    }
-
-    const ignoreProps = [
-      'theme',
-      'children',
-      'styles',
-      'makeStyles',
-      'elementRef'
-    ]
-
-    Object.keys(View.propTypes)
```

Comment:
Shouldn't we keep these tests? We can now iterate through `View.allowedProps` instead of `Object.keys(View.propTypes)`.

Edit: I can see that you made tests for the utils, which is awesome and is probably enough, but I'm afraid to delete tests :) These test it from a little different approach after all.

Reply:
I've re-added these kind of tests.